.goreleaser.yaml

```diff
@@ -33,6 +42,31 @@ builds:
         -X "github.com/grafana/pyroscope/pkg/util/build.Revision={{ .ShortCommit }}"
         -X "github.com/grafana/pyroscope/pkg/util/build.BuildDate={{ .CommitDate }}"
     id: pyroscope
+  - env:
```

Comment:
optional: YAML anchors and aliases are a good tool for reducing duplication and improving readability. Up to you if you think it's worthwhile here.

Reply:
I thought about this, but after seeing https://github.com/goreleaser/goreleaser-action/issues/397 I decided against it, did not actually test it if it's still affected.